this prints a log line for every single client call..
for applications that call this, it ends up calling this a hundred times a second and printing the same log line
--
diff --git a/src/wl_window.c b/src/wl_window.c
index 53cbd33..cb2a2fb 100644
--- a/src/wl_window.c
+++ b/src/wl_window.c
@@ -1959,16 +1959,16 @@ void _glfwPlatformGetWindowPos(_GLFWwindow* window, int* xpos, int* ypos)
     // A Wayland client is not aware of its position, so just warn and leave it
     // as (0, 0)
 
-    _glfwInputError(GLFW_PLATFORM_ERROR,
-                    "Wayland: Window position retrieval not supported");
+    //_glfwInputError(GLFW_PLATFORM_ERROR,
+    //                "Wayland: Window position retrieval not supported");
 }
 
 void _glfwPlatformSetWindowPos(_GLFWwindow* window, int xpos, int ypos)
 {
     // A Wayland client can not set its position, so just warn
 
-    _glfwInputError(GLFW_PLATFORM_ERROR,
-                    "Wayland: Window position setting not supported");
+    //_glfwInputError(GLFW_PLATFORM_ERROR,
+    //                "Wayland: Window position setting not supported");
 }
 
 void _glfwPlatformGetWindowSize(_GLFWwindow* window, int* width, int* height)
